neptune/management/internal/utils.py

```diff
@@ -24,13 +24,13 @@
 from neptune.patterns import PROJECT_QUALIFIED_NAME_PATTERN
 
 
-def normalize_project_name(name: str, workspace: Optional[str] = None):
+def extract_project_and_workspace(name: str, workspace: Optional[str] = None):
```

Comment:
Please use this function as well in `management.api` instead of `project_identifier.split("/")[0]` in two places.